src/handlers/movies/getMovies.ts

```diff
@@ -0,0 +1,79 @@
+import { getAll as getAllMovies } from '../../models/Movie';
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import getAll from '../../models/Genre/getAll';
+
+export const handler: APIGatewayProxyHandler = async (event) => {
+	try {
+		const queryParams: {
+			page?: number;
+			limit?: number;
+			genre?: string;
+			search?: string;
+			ids?: string[];
+		} = {};
+
+		const { queryStringParameters } = event;
```

Comment:
Totally fine and cleaner way to remove an extra if enclosure
```suggestion
		const { page, limit, genre, search } = event?.queryStringParamters ?? {};
```